runtests.py

```diff
@@ -249,6 +248,38 @@ def update_openmp_extension(ext):
 
     return EXCLUDE_EXT
 
+def update_cpp11_extension(ext):
+    """
+        update cpp11 extensions that will run on versions of gcc >4.8
+    """
+    gcc_version = get_gcc_version()
+    if gcc_version is not None:
+        if float(gcc_version) > 4.8:
+            compile_flags, link_flags = flags
+            ext.extra_compile_args.extend("-std=c++11")
+        return ext    
+    return EXCLUDE_EXT
+
+
+def get_gcc_version():
+    """
+        finds gcc version using Popen
+    """
+    matcher = re.compile(r"gcc version (\d+\.\d+)").search
+    try:
+        p = subprocess.Popen([cc, "-v"], stderr=subprocess.PIPE, env=env)
```

Comment:
`cc` is undefined. You need to also move over the initial setup from the original function (CC/env/etc.).